[2023-09-13T07:38:28.413+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-25T00:00:00+00:00 [queued]>
[2023-09-13T07:38:28.440+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-25T00:00:00+00:00 [queued]>
[2023-09-13T07:38:28.445+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-13T07:38:28.476+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): extract_data> on 2023-08-25 00:00:00+00:00
[2023-09-13T07:38:28.488+0000] {standard_task_runner.py:57} INFO - Started process 91 to run task
[2023-09-13T07:38:28.545+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'extract_data', 'scheduled__2023-08-25T00:00:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpjpv9iakw']
[2023-09-13T07:38:28.583+0000] {standard_task_runner.py:85} INFO - Job 25: Subtask extract_data
[2023-09-13T07:38:28.777+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data.extract_data scheduled__2023-08-25T00:00:00+00:00 [running]> on host 190733975a8f
[2023-09-13T07:38:29.146+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-08-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-25T00:00:00+00:00'
[2023-09-13T07:38:29.468+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-13T07:38:29.484+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=extract_data, execution_date=20230825T000000, start_date=20230913T073828, end_date=20230913T073829
[2023-09-13T07:38:29.542+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-13T07:38:29.559+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-15T11:31:08.749+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-25T00:00:00+00:00 [queued]>
[2023-09-15T11:31:08.757+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-25T00:00:00+00:00 [queued]>
[2023-09-15T11:31:08.757+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-15T11:31:08.784+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): extract_data> on 2023-08-25 00:00:00+00:00
[2023-09-15T11:31:08.794+0000] {standard_task_runner.py:57} INFO - Started process 335 to run task
[2023-09-15T11:31:08.809+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'extract_data', 'scheduled__2023-08-25T00:00:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpmgqg_iuj']
[2023-09-15T11:31:08.812+0000] {standard_task_runner.py:85} INFO - Job 17: Subtask extract_data
[2023-09-15T11:31:08.877+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data.extract_data scheduled__2023-08-25T00:00:00+00:00 [running]> on host 4af8904e631c
[2023-09-15T11:31:08.984+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-08-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-25T00:00:00+00:00'
[2023-09-15T11:31:09.267+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-15T11:31:09.277+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=extract_data, execution_date=20230825T000000, start_date=20230915T113108, end_date=20230915T113109
[2023-09-15T11:31:09.332+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-15T11:31:09.349+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
